packages/components/video/VideoCard.tsx

```diff
@@ -119,6 +122,21 @@ export const VideoCard: React.FC<{
               {prettyMediaDuration(video.videoFile.videoMetadata?.duration)}
             </BrandText>
           </View>
+
+          {video?.location && (
+            <View style={positionButtonBoxStyle}>
+              <LocationButton
+                onPress={() =>
+                  video?.location &&
```

Comment:
dont need to check video?.location again, we have already that value when reaching this code

Reply:
I don't know why Typescript gives an error if I don't recheck `video?.location` here.

```Argument of type '{ lat: number; lng: number; } | [number, number] | undefined' is not assignable to parameter of type '{ lat: number; lng: number; } | [number, number]'.
  Type 'undefined' is not assignable to type '{ lat: number; lng: number; } | [number, number]'.ts(2345)```